.github/workflows/unit_tests.yml

```diff
@@ -41,7 +41,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        image: ["postgres:10", "postgres:14"]
+        image: ["postgres:11", "postgres:16"]
```

Comment:
To address @johha's concern we could append 11 and 16 to the list instead of replacing it right away.  The precedent for this is that the current concourse based pipeline still uses mysql 5.7 and not 8.

Same thing could be done with mysql 8.2 below

Reply:
Ok i readded psql 10 and added mysql 8.2 as new job.
This leads to 6 actions beeing executed per PR now so ~30% increas in CI cost is likely but i think we`re fine with this temporarily until we adopt the release pipelines :)